src/man/sssd-ifp.5.xml

```diff
@@ -133,7 +137,9 @@
                             It is possible to add another attribute to
                             this set by using <quote>+attr_name</quote>
                             or explicitly remove an attribute using
-                            <quote>-attr_name</quote>. For example, to
+                            <quote>-attr_name</quote>. Added attributes
+                            will be made available in the
+                            <quote>extraAttributes</quote> field. For example, to
```

Comment:
Hi,

maybe "array" would be better here instead of "field" because on the DBus level it is labeled as array:
```
      dict entry(
         string "extraAttributes"
         variant             array [
               dict entry(
                  string "objectSIDString"
                  array [
                     string "S-1-5-21-2982822198-506007722-229075370-500"
                  ]
               )
            ]
      )
```
bye,
Sumit

Reply:
Agreed, fixed.

Reply:
Thank you.